src/renderer/components/Transaction/TransactionForm/TransactionFormTransfer.vue

```diff
@@ -244,15 +244,11 @@ export default {
     vendorFieldLabel () {
       return `${this.$t('TRANSACTION.VENDOR_FIELD')} - ${this.$t('VALIDATION.MAX_LENGTH', [64])}`
     },
-    vendorFieldError () {
-      if (this.vendorFieldIsInvalid) {
-        return this.$t('VALIDATION.TOO_LONG', [this.$refs.vendorField.label])
+    vendorFieldHelperText () {
+      if (this.form.vendorField.length === 64) {
```

Comment:
Instead of hardcoding `64` here:

1. add that value to each network configuration of `config/networks/*.json`
```json
"vendorField": {
  "maxLength": 64
}
```

2. add a new key on `config/index.js`:

```js
exports.VENDOR_FIELD = {
  defaultMaxLength: 64
}
```